frontend/src/views/web/auth/get-token.html

```diff
@@ -0,0 +1,29 @@
+<section class="text-center pd-20 width-1200">
+    <div class="row ht-wt-100p">
+        <div class="col s12 m12">
+            <div class="ev-card-body">
+                <form name="getTokenForm" ng-submit="profile.getAuthToken(getTokenForm.$valid)">
+                    <div class="pass-title">Your Auth Token.</div>
```

Comment:
Please change `Your Auth Token.` to `API Authentication Token`

Reply:
@RishabhJain2018 Done!